ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -11,6 +11,11 @@ export AWS_SHARED_CREDENTIALS_FILE=$cluster_profile/.awscred
 export AZURE_AUTH_LOCATION=$cluster_profile/osServicePrincipal.json
 
 echo "Deprovisioning cluster ..."
+if [ ! -f "${SHARED_DIR}/metadata.json" ]; then
```

Comment:
nit: always prefer `[[` over `[` and `-s` over `-f` unless you want an _empty_ JSON to work ;)

```bash
if [[ ! -s "${SHARED_DIR}/metadata.json" ]]; then
```

Reply:
ack, done.